teams/vsp/teams/tools/backend/external-service-monitoring/README.md

```diff
@@ -0,0 +1,43 @@
+# VSP Toolkit | SLO Tools
+
+## External Service Monitoring
+
+This folder contains tools used for managing external service monitoring reports.
+
+### Generate External Service Performance report
+
+These instructions explain how to use the `Selenium::WebDriver` based script to capture snapshots of each External Service dashboard.
+
+#### Prerequisites:
+
+These instructions and corresponding script assume the following:
+
+- `ruby --version` should be `2.3.x` or greater
+- `convert --version` should be `Version ImageMagick 6.9.x` or greater [install instructions](https://imagemagick.org/script/download.ph://imagemagick.org/script/download.php)
```

Comment:
This link is malformed, looks like it needs to be: https://imagemagick.org/script/download.php